### DISCUSSION TITLE
How many bones do sharks have?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#564] [Galaxy Trivia] [answered]
🦈🦈🦈

**Comment by peterbaccaro** [comment, marked as answer]
**Sharks do not have any bones**, they are a special type of fish known as "elasmobranchs", which translates into fish made of cartilaginous tissues—the clear gristly stuff that your ears and nose tip are made of. 

This category also includes rays, sawfish, and skates. Their cartilaginous skeletons are much lighter than true bone and their large livers are full of low-density oils, both helping them to be buoyant. 

Even though sharks don't have bones, they still can fossilize. As most sharks age, they deposit calcium salts in their skeletal cartilage to strengthen it. The dried jaws of a shark appear and feel heavy and solid; much like bone. These same minerals allow most shark skeletal systems to fossilize quite nicely. The teeth have enamel so they show up in the fossil record too.

**Comment by thuongtruong1009** [comment]
Sharks are cartilaginous fish and their skeletons are made of cartilage, not bones like most other fish and animals. Cartilage is a tough, flexible tissue that provides support and structure but is more lightweight and flexible than bone. Therefore, sharks do not have bones like humans and other vertebrates.
